app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -751,6 +758,21 @@ class HomeFragment : Fragment() {
         lifecycleScope.launch(IO) {
             requireComponents.reviewPromptController.promptReview(requireActivity())
         }
+
+        if (FeatureFlags.showWallpapers) {
+            binding.wordmark.setOnClickListener {
+                val wallpaper = context.settings().toggleWallpaper()
+                updateWallpaper(wallpaper, context)
+            }
+        }
+    }
+
+    private fun updateWallpaper(wallpaper: Wallpapers, context: Context) {
```

Comment:
Can't we access the `context` within the function without passing it in as a param?